Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour
+{
+    private bool isPlayerInTrigger;
+ 
+    //private void OnTriggerStay2D(Collider2D collision)
+    //{
+    //    if (collision.CompareTag("Player"))
+    //    {
+    //        if (Input.GetButtonDown("Activate"))
+    //        {
+    //            Debug.Log("Player Activated Door!");
+    //        }
+    //    }
+    //}
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
+            isPlayerInTrigger = true;
+        }
+    }
+
+    private void OnTriggerExit2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
+            isPlayerInTrigger = false;
+        }
+    }
+
+    private void Update()
+    {
+        if (Input.GetButtonDown("Activate") && isPlayerInTrigger)
+        {
+            Debug.Log("Player Activated Door!");
+            SceneManager.LoadScene("Level1");
```

Comment:
I would recommend using Scene Indexes for calling a scene, this entails going into your unity build settings and adding scenes in a specific order to your build and then it allows for the calling of scenes by their unity scene index number. In the main menu video on canvas, the recorder and commentator uses this if you need an example.